_posts/2020-09-20-pulp-3.7.0-is-generally-available.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Pulp 3.7.0 is Generally Available!
+author: Melanie Corr
+tags:
+  - release
+---
+
+The Pulp team are pleased to announce the release of Pulp 3.7.0!
+
+
+## Export enhancements
+
+There have been a number of enhancements to the export process:
+
+* A progress report has been added to export tasks. [#6541](https://pulp.plan.io/issues/6541)
+
+* Improved handling of failed exports. Previously, when an export failed, the export's tarball was left on the filesystem. This has now been cleaned up so that nothing remains and there is a clean slate. [#7246](https://pulp.plan.io/issues/7246)
+
+* A `TOC_info` file has been added to point to the location of the TOC file during the export process. [#7221](https://pulp.plan.io/issues/7221)
+
+## Managing checksums
+
+`ALLOWED_CONTENT_CHECKSUMS` have been introduced so that users can specify and limit unwanted checksums in their environment. Note that plugins might not yet support `ALLOWED_CONTENT_CHECKSUMS` and that users should check plugin's release notes/docs for support before setting CONTENT_CHECKSUMS. [#5216](https://pulp.plan.io/issues/5216)
+
+## Enabling automatic file deletion for Filefield
+
+`FileField` has been enhanced with Django cleanup handlers so that files can be deleted from the `FileField`. With this release, when the delete() method is  evoked on the corresponding `FileField` model, automatic cleanup occurs. [#7316](https://pulp.plan.io/issues/7316)
+
+## Managing the Installation of Optional Dependences
+
+To make it easier to install optional dependencies such as S3 and Promethious support, as well as some test dependencies, an [**extras_require**](https://setuptools.readthedocs.io/en/latest/setuptools.html#declaring-extras-optional-features-with-their-own-dependencies) argument has been added. [#6844](https://pulp.plan.io/issues/6844)
+
+## REST API Bug Fixes
+
+For a full list of bug fixes, see the [bug fixes](https://docs.pulpproject.org/pulpcore/changes.html#bugfixes) section of the changelog.
+
+## Plugin API & Documentation
+
+The Plugin API has also been updated with enabling of automatic deletion of files in [#7316](https://pulp.plan.io/issues/7316) and [adding `pulpcore.app.tasks.repository.add_and_remove` to `pulpcore.plugin.tasking`](https://pulp.plan.io/issues/7351).
```

Comment:
The first part of this sentence I find confusing. I would probably just call this out in the django-cleanup section (see my previous comment) and leave it at that.